config/schedule.rb

```diff
@@ -9,30 +13,36 @@ def stagger(hour)
 
 # bi-weekly harvest at 5pm in qa, on the 8th and 23rd of the month
 every "0 17 7,23 * *", roles: [:harvester_qa] do
-  rake 'harvest:all_authors_update'
+  set :check_in, Settings.honeybadger_checkins.harvest_all_authors
+  rake_hb 'harvest:all_authors_update'
 end
 
 # every three day harvest at 5pm in prod, don't overlap with qa
 every "0 17 1,5,9,13,17,21,25,29 * *", roles: [:harvester_prod] do
-  rake 'harvest:all_authors_update'
+  set :check_in, Settings.honeybadger_checkins.harvest_all_authors
+  rake_hb 'harvest:all_authors_update'
 end
 
 # poll cap for new authorship information nightly at 4am-ish in prod, qa and dev
 every 1.day, at: stagger(4), roles: [:harvester_dev, :harvester_qa, :harvester_prod] do
-  rake 'cap:poll[1]'
+  set :check_in, Settings.honeybadger_checkins.cap_poll
+  rake_hb 'cap:poll[1]'
 end
```

Comment:
I guess it's no big deal if this task winds up trying to check in on dev and QA? It'll be a no-op?